virt/install/installing-virt.adoc

```diff
@@ -15,7 +15,6 @@ to subscribe to and deploy the {VirtProductName} Operators.
 * {product-title} {product-version}
 
 :FeatureName: {VirtProductName}
```

Comment:
Should this also remove the FeatureName attribute? @aburdenthehand ?

Same below in other places.

Reply:
Right you are. Thanks for the catch.
#23978 takes care of it